app/src/androidTest/java/com/sdp/movemeet/view/map/MainMapFragmentTest.java

```diff
@@ -217,11 +143,6 @@ public void mainMapFragment_userClickingOnMapAddsNewActivity() throws UiObjectNo
         uiDevice.click(200, 400);
     }
 
-    @After
-    public void after() {
```

Comment:
Why you remove the logout at the end of the test ? Isn't it necessary for the good progress of all the tests?

Reply:
With this, I had to add a login before every map test, which was redundant as most of those tests don't even need the user to be logged in. Since the following tests are the security tests, that always check that the user is not logged (and if he is, unlogged him), that doesn't create any problem. But we might have to change it  afterwards anyway, when we will mock the authentication.